app/com/github/pedrovgs/dotto/core/DottoApp.scala

```diff
@@ -0,0 +1,26 @@
+package com.github.pedrovgs.dotto.core
+
+import cats.free.Free
+import com.github.pedrovgs.dotto.core.algebra.dotto._
+
+/**
+  * Declares the two programs Dotto implements.
+  */
+object DottoApp {
+
+  def enqueueMessageProgram: Message => Free[Instruction, Message] = {
```

Comment:
If you are using a value to capture this function you may just use `val` instead of def since `enqueMessageProgram` points to a function. Alternatively you may declare this as `def enqueMessageProgram(m : Message) : Free[Instruction, Message] = ...`